cypress/support/po/home.po.ts

```diff
@@ -0,0 +1,6 @@
+export const getInput = () => cy.get('[data-testid="add-task-input"]');
+export const getInputField = () => cy.get('[data-testid="add-task-input"] input');
```

Comment:
The Cypress `.clear()` method don't work on `<TextField />` component since it is wrapped by a `<div>`.
This is the way I found to erase the field data before each input test.
Let me know if it is not the best practice.